--- misc/fts.c	2020-12-12 12:20:36.000000000 -0800
+++ misc/fts.c	2020-12-12 12:21:51.000000000 -0800
@@ -63,6 +63,7 @@
 #endif
 #if defined(__APPLE__)
 #   define __errno_location()	(__error())
+#   define stat64		stat
 #endif

 #include "system.h"
--- lib/rpmrc.c	2020-12-12 15:35:40.000000000 -0800
+++ lib/rpmrc.c	2020-12-12 15:37:11.000000000 -0800
@@ -1039,6 +1039,8 @@
 	    strcpy(un.machine, "i386");
 #elif defined(__x86_64__)
 	    strcpy(un.machine, "x86_64");
+#elif defined(__arm64__)
+            strcpy(un.machine, "aarch64");
 #else
 	    #warning "No architecture defined! Automatic detection may not work!"
 #endif 
